utils/bluez-tools/files/bt-agent.init

```diff
@@ -9,29 +11,33 @@ SERVICE_PID_FILE=/var/run/$SERVICE_NAME/pid
 DESC="Bluetooth pairing agent"
 DAEMON=/usr/bin/$SERVICE_NAME
 
-
-# Exit if the package is not installed
-[ -x "$DAEMON" ] || exit 0
-
 AGENT_PIN_FILE=/etc/bt-agent.cfg
```

Comment:
Pid files are located in `/var/run/`, so please change this to `/var/run/bt-agent.pid`

Reply:
this is the pin file needed by bt-agent not a pid file

Reply:
Ok, I read that wrong.
But since we are on an embedded system, the write accesses to the NAND should be reduced.
Therefore it is a must that we move the generated file to /tmp and not keep it in /etc.

Reply:
Alright that seems like a logical thing to do. I will update this and the other stuff as soon as I got some spare time.